scienceplots/__init__.py

```diff
@@ -1,11 +1,8 @@
 import os  # pathlib.Path.walk not available in Python <3.12
 import matplotlib.pyplot as plt
 
-import scienceplots
-
 # register the bundled stylesheets in the matplotlib style library
-scienceplots_path = scienceplots.__path__[0]
-styles_path = os.path.join(scienceplots_path, 'styles')
+styles_path = os.path.join(__file__, 'styles')
```

Comment:
`__file__` is resolving to `scienceplots/__init__.py`, so `styles_path = os.path.join(__file__, 'styles')`  resolves to `scienceplots/__init__.py/styles`, which is undesirable.

Reply:
```suggestion
styles_path = os.path.join(os.path.dirname(__file__), 'styles')
```